pkg/server/container_status.go

```diff
@@ -70,6 +74,20 @@ func toCRIContainerStatus(container containerstore.Container, spec *runtime.Imag
 			reason = errorExitReason
 		}
 	}
+	if verbose {
+		info = make(map[string]string)
+		m, _ := json.Marshal(status.Pid)
+		info["pid"] = string(m)
+		m, _ = json.Marshal(meta.Config)
+		info["container config"] = string(m)
+		m, _ = json.Marshal(status.Removing)
+		info["removing state"] = string(m)
+		oldSpec, err := container.Container.Spec(ctx)
```

Comment:
Spec will be pretty long, I feel like this is too detailed now.
We could include information useful separately instead of including the whole spec directly.

Reply:
Spec has lots of goodies in it, yes.  And some repeats vs the container config information. Looks good in json mode and lets someone verify / see what their generated spec was. Spec will likely change over time and be different per container options.  

It might be easier to "filter" out the parts that we don't want.

Let's put it this way if we pick and choose one at a time there will be a lot of PRs to keep coming back to add just one more item.  Then someone will ask .. hey can't you just dump the spec so I can copy paste it and test in runc?

Reply:
@mikebrow I don't think `docker` user want oci spec, that' also why `docker inspect` return oci spec. It's too detailed for docker user, we are trying to keep similar user experience with docker.

Reply:
I thought the idea was a debugging use case for containers under kube, esp. where the runtime is runc based and thus requiring an OCI runtime specification, which is now supported by containerd. The actual runtime spec used to generate those containers is a pretty important part of that debugging task. We could cut it up by removing parts or pick parts one at a time, but my concern is missing parts of the spec that are important to the tax of identifying and fixing bugs. We should discuss.